File: FirstApp/app/src/main/java/com/example/firstapp/MainActivity.java
package com.example.firstapp;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Find button by ID
        Button buttonShowMessage = findViewById(R.id.buttonShowMessage);

        // Set click listener
        buttonShowMessage.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // Show Toast message
                Toast.makeText(MainActivity.this, "Button Clicked!", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: app/src/main/java/com/example/firstapplication/MainActivity.kt
package com.example.myfirstapp

import android.R
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    protected override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_list_item)

        // Find button by ID
        val buttonShowMessage: Button = findViewById(R.id.button1)

        // Set click listener
        buttonShowMessage.setOnClickListener { // Show Toast message
            Toast.makeText(this@MainActivity, "Button Clicked!", Toast.LENGTH_SHORT).show()
        }
    }
}
